manpages/lumina-config.8

```diff
@@ -1 +1,25 @@
-Dummy file
+.Dd November 3, 2017
+.Dt LUMINA-CONFIG 8
+.Os 1.3.3
+
+.Sh NAME
+.Nm lumina-config
+.Nd used to modify different desktop settings.
```

Comment:
"used to" is unnecessary - make the sentence active: "modifies desktop settings"